components/brandprofile/ColorShowcase.tsx

```
@@ -0,0 +1,196 @@
+    <motion.div
+      whileHover={{ scale: 1.05 }}
+      className={`group relative ${className}`}
+    >
+      <button
+        onClick={() => handleCopy(color.hex)}
+        onMouseEnter={() => setShowTooltip(true)}
+        onMouseLeave={() => setShowTooltip(false)}
+        className="w-full focus:outline-none focus:ring-2 focus:ring-primary-500 focus:ring-offset-2 rounded-lg"
+        aria-label={`Copy ${color.name} color ${color.hex}`}
+        tabIndex={0}
+      >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)